language/stdlib/transaction_scripts/add_to_script_allow_list.move

```diff
@@ -2,6 +2,28 @@ script {
 use 0x1::LibraTransactionPublishingOption;
 use 0x1::SlidingNonce;
 
+/// # Sender Preconditions
+/// The sending account (`lr_account`) must be the Libra Root account. The script allow
+/// list must not already hold the script `hash` being added.
+/// # Accounts Affected
+/// The Libra Root account is affected with normal transaction effects (sequence number,
```

Comment:
Do we want to say this, since it's true of every tx?

Reply:
Not entirely sure,  think it's something that we should ask tomorrow morning on the call, it would be nice if we didn't!